src/components/AppNotification/__snapshots__/AppNotification.test.ts.snap

```diff
@@ -53,7 +53,7 @@ exports[`<AppNotification /> LightColor story renders snapshot 1`] = `
           class="button-group button-group--horizontal-align-left"
         >
           <button
-            class="button button--layout-none button--secondary button--lg button--variant-default"
+            class="button button--layout-none button--secondary button--lg button--size-lg button--variant-default"
```

Comment:
Confirming we want both button--lg and button--size-lg. Seems like yes based on the description.